docs/api/crossplaneio/crossplane/database-crossplane-io-v1alpha1.md

```diff
@@ -67,7 +67,7 @@ Appears in:
 
 Name | Type | Description
 -----|------|------------
-`engineVersion` | string | EngineVersion specifies the desired PostgreSQL engine version, e.g. 9.6.
+`engineVersion` | string | EngineVersion specifies the desired PostgreSQL engine version. 9.6 or 11 are allowed.
```

Comment:
These docs are auto-generated using https://github.com/negz/crossdocs, so this should reflect the code comment on the `engineVersion` field. Unfortunately, it is a bit difficult for to setup `crossdocs` currently, so I see the following options here:
- You could remove this change to the docs and opt for the existing comment (it isn't wrong currently, but your update is an improvement IMO).
- You could setup `crossdocs` (I am more than happy to help here), and you could modify the code comment, then regenerate the docs here.
- You could remove this change, then I would be happy to follow-up with a PR to change the code comment and regenerate the docs. I could add you as an approver to make sure it matches your expectations.

This is a poor part of the development workflow right now and we need to move on getting a fix for it (related issue:  https://github.com/crossplaneio/crossplane/issues/1166). Apologies for the complications it is currently causing!